public/404.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```

Comment:
I don't think this folder is going to be used in the future (/public). Given how `nx build` works, the dist files are placed somewhere else. In the firebaserc.json you have to point to the real public folder. in this case I believe the path is `/dist/apps/event-manager-web-client/index.html`